src/polymer-language-server.ts

```diff
@@ -106,12 +106,46 @@ connection.onInitialize((params): InitializeResult => {
   };
 });
 
+async function mapAllInitialDocuments(editorService: EditorService) {
+  for (const document of documents.all()) {
+    const localPath = getWorkspacePathToFile(document);
+    if (localPath) {
+      await editorService.fileChanged(localPath, document.getText());
+    }
+  }
+  await reportErrors(editorService, connection);
+}
+
 // The content of a text document has changed. This event is emitted
 // when the text document is first opened or when its content has changed.
 documents.onDidChangeContent((change) => {
-  scanDocument(change.document, connection);
+  return handleErrors(handleChangedDocument(change.document), undefined);
 });
 
+async function handleChangedDocument(document: TextDocument) {
+  if (editorService) {

```

Comment:
This is why #62 manages the editor service instance properly. This was just paranoia on my part.